jkube-kit/config/resource/src/main/java/org/eclipse/jkube/kit/config/resource/JKubeAnnotations.java

```diff
@@ -36,21 +36,36 @@ public enum JKubeAnnotations {
     SCM_TAG("scm-tag"),
     SCM_URL("scm-url"),
 
-    TARGET_PLATFORM("target-platform");
+    TARGET_PLATFORM("target-platform"),
+    ICON_URL("iconUrl");
+
+    private static final String JKUBE_ANNOTATION_PREFIX = "jkube.eclipse.org";
+    /**
+     * @deprecated in favor of <code>jkube.eclipse.org</code>
+     */
+    @Deprecated
+    private static final String DEPRECATED_JKUBE_ANNOTATION_PREFIX = "jkube.io";
 
     private final String annotation;
 
     JKubeAnnotations(String anno) {
-        this.annotation = "jkube.io/" + anno;
+        this.annotation = "/" + anno;
     }
 
-    public String value() {
-        return annotation;
+    public String value(boolean useDeprecatedPrefix) {
```

Comment:
To me it would be more elegant to set the `useDeprecatedPrefix` with a `with` method
```
JKubeAnnotations.GIT_COMMIT.withUseDeprecatedPrefix(true).value();
```
default with `useDeprecatedPrefix` to false:
```
JKubeAnnotations.GIT_COMMIT.value();
```
at the moment, when reading the code, it is not clear what is passed to `value(arg)`

Reply:
IMHO I'd simplify this, just provide two methods (no need for a public static method that is not reused anywhere?), a simple value with the boolean parameter that overloads and calls the value() method with `false` (default),

Reply:
I checked and `getAnnotationPrefix` method seems to be getting used in [AutoTLSEnricher](https://github.com/rohanKanojia/jkube/blob/pr/issue1273/jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/openshift/AutoTLSEnricher.java#L181) to determine jkube prefixes  in mount points:

https://github.com/eclipse/jkube/blob/c9de4cce64bf762cc1f56f95bfcd1af61eccdfc2/jkube-kit/enricher/generic/src/main/java/org/eclipse/jkube/enricher/generic/openshift/AutoTLSEnricher.java#L55

Reply:
I've created a separate issue to handle this case https://github.com/eclipse/jkube/issues/2068 . I've moved this static method to a utility class